### PR TITLE
Feat(UI): calc component

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feat(UI): calc component
In this feature update:
Delete all the boilerplate from CRA (text, images, styles).

- Create a directory called components.
- Inside components, create a new ```Calculator.js and calculator.css``` file.
- In Calculator.js, create a React component that matches the [proposed design](https://github.com/microverseinc/curriculum-react-redux/raw/main/math-magicians/images/calculator.png).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
-function App() {
+export default function App() {
```
Comment (Davidosky007) [inline src/App.js]: * Kindly Use class-based components here as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.).

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @blessedjasonmwanza  👋, 

Good job so far! 💪 
* No linter errors ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
* Proper use of git-flow ✔️ 
## There are some issues that you still need to work on to go to the next project but you are almost there!
almost-gif


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (blessedjasonmwanza) [reply src/App.js]: Hello @Davidosky007 ,

Kindly note: I used a class-based component for the Calculator, as per advised.
Kindly check the Calculator Component.

Comment (Davidosky007) [reply src/App.js]: * Kindly note that  All components should be class-based as in the next project you will be asked to refactor all back to functional components 
image

Comment (blessedjasonmwanza) [reply src/App.js:3]: 👨🏽‍💻  I'll make the proposed changes right away 😃 . Thanks a billion for this insightful review @Davidosky007  🤓 .

Comment (blessedjasonmwanza) [reply src/App.js:3]: 👨🏽‍💻  I'll make the proposed changes right away 😃 . Thanks a billion for this insightful review @Davidosky007  🤓 .

Comment (blessedjasonmwanza) [reply src/App.js]: 👨🏽‍💻  I'll make the proposed changes right away 😃 .

> Thanks a billion for this insightful review @Davidosky007  🤓 .

Comment (blessedjasonmwanza) [reply src/App.js]: Actioned ✅  .On to requesting for another review.

> HappyCoding/>

**Comment by omar-labana** [comment]
Hi @blessedjasonmwanza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — COMMENTED** [review]
## **Hi @blessedjasonmwanza,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**